python/cudf/cudf/_libxx/table.pyx

```diff
@@ -273,3 +252,24 @@ cdef mutable_table_view _make_mutable_table_view(columns) except*:
         mutable_column_views.push_back(col.mutable_view())
 
     return mutable_table_view(mutable_column_views)
+
+cdef columns_from_ptr(
+        unique_ptr[table] c_tbl,
+    ):
```

Comment:
Indentation of this whole function looks off and there should be a line break at the end of the file :)

Reply:
Fixed